File: Data Structures & Algorithms/is-anagram/submission-6.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t) :
            return False
    
        Count = {}

        for i in range(len(s)):
            Count[s[i]] = Count.get(s[i],0)+1
            Count[t[i]] = Count.get(t[i],0)-1

        for c in Count.values():
            if c != 0:
                return False
            

        return True

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-6.java
class Solution {
    public int longestConsecutive(int[] nums) {
        HashSet<Integer> set = new HashSet<>();

        for(int num : nums){
            set.add(num);
        }
        int count = 0;

        for(int num : set){
            if(set.contains(num-1)) continue;

            int helper = 1;
            int copy = num;
            while(set.contains(copy+1)){
                helper++;
                copy++;
            }

            count = Math.max(count, helper);
        }

        return count;
    }
}
